aidants_connect_web/templates/public_website/politique_confidentialite.html

```diff
@@ -0,0 +1,191 @@
+      </ul>
+      <h2 class="fr-mt-4w">Quelles sont les données que nous traitons ?
+      </h2>
+      <p class="fr-mb-1w">Aidants Connect traite les données à caractère personnel suivantes :</p>
+      <ul class="fr-pl-2w">
+        <li><strong>Données d'identification de l'usager :</strong> nom, prénom, date et lieu de naissance, genre, adresse e-mail, alias technique FranceConnect, alias technique Aidants Connect ;</li>
+        <li><strong>Données d'identification des aidants habilités par les structures :</strong> nom, prénom, adresse e-mail professionnelle, fonction ;</li>
+        <li><strong>Données relatives au mandat :</strong> nom et prénom de l'usager, nom et prénom de l'aidant.</li>
+      </ul>
+      <h2 class="fr-mt-4w">Qu'est-ce qui nous autorise à traiter ces données ?</h2>
+      <p class="fr-mb-1w">Aidants Connect traite des données à caractère personnel en se basant sur :</p>
+      <ul class="fr-pl-2w">
+        <li>L'exécution d'une mission d'intérêt public ou relevant de l'exercice de l'autorité publique dont est investi le responsable de traitement au sens de l'article 6-1 e) du RPGD.</li>
+      </ul>
+      <h2 class="fr-mt-4w">Pendant combien de temps conservons-nous ces données ?</h2>
+      <div class="fr-table fr-table--bordered fr-table--layout-fixed">
+        <table>
+          <thead>
+            <tr>
+              <th scope="col">Type de données</th>
+              <th scope="col">Durées de conservation</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><strong>Données d'identification de l'usager</strong></td>
+              <td>3 ans en base active après l'expiration de leur dernier mandat et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+            <tr>
+              <td><strong>Données d'identification des aidants habilités par les structures</strong></td>
+              <td>3 ans en base active à compter du dernier contact avec l'aidant et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+            <tr>
+              <td><strong>Données relatives au mandat</strong></td>
+              <td>3 ans en base active à compter de l'expiration du mandat et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+          </tbody>
+        </table>
+      </div>
+      <h2 class="fr-mt-4w">Quels droits avez-vous ?</h2>
+      <p class="fr-mb-1w">Vous disposez :</p>
+      <ul class="fr-pl-2w fr-mb-1w">
+        <li>D'un droit d'information et d'un droit d'accès à vos données ;</li>
+        <li>D'un droit de rectification ;</li>
+        <li>D'un droit d'opposition ;</li>
+        <li>D'un droit à la limitation du traitement.</li>
+      </ul>
+      <p>Pour les exercer, contactez-nous à : <a href="mailto:contact@aidantsconnect.beta.gouv.fr" class="fr-link" target="_blank">contact@aidantsconnect.beta.gouv.fr</a></p>
```

Comment:
```suggestion
      <p>Pour les exercer, contactez-nous à : {% mailto_href AC_CONTACT_EMAIL %}</p>
```

Faudra `{% load ac_common %}` aussi.

Reply:
👍